core/src/main/java/org/stellar/anchor/util/JsonLog.java

```diff
@@ -0,0 +1,14 @@
+package org.stellar.anchor.util;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class JsonLog {
+  @Getter @Setter Integer time;
+
+  @Getter @Setter String source;
+
+  @Getter @Setter String index;
+
+  @Getter @Setter EventObjectJson event;
+}
```

Comment:
Since you're only using `JsonLog` (and `EventObjectJson`) inside the tests, maybe it's better to declare it in the test itself?

Actually, you could delete these classes and use a Map to parse the json and test its values, similar to what we do here:
https://github.com/stellar/java-stellar-anchor-sdk/blob/f82985c0ce81df8aeee6fd3c95db30a224a7e69e/integration-tests/src/main/kotlin/org/stellar/anchor/platform/Sep31Client.kt#L19-L20

In general, I don't think we should add new models to the project `src` if they are only meant to be used for tests.